src/preprocess/index.ts

```diff
@@ -50,6 +50,10 @@ function interpolateHandlebarsString(templateString: string, env: object, errorC
 
 export type SHA256Digest = string;
 
+export type PreprocessOptions = {
+  omitMetadata?: boolean
```

Comment:
I think it's worth looking into ways of making the `metadata` deterministic so we can get rid of this logic.